pyvista/utilities/helpers.py

```diff
@@ -960,7 +960,7 @@ def wrap(dataset):
         try:
             return pyvista._wrappers[key](dataset)
         except KeyError:
-            logging.warning(f'VTK data type ({key}) is not currently supported by pyvista.')
+            warnings.warn(f'VTK data type ({key}) is not currently supported by pyvista.')
```

Comment:
This one seems like we should raise an error instead.  I don't see how a user would be able to proceed after this point without handling it.

Reply:
I was about to say that we rely on this behaviour in `readers.py`, but actually that's just another raise, just like you suggest here
https://github.com/pyvista/pyvista/blob/31f4886cd42742182be56e16b5cb26af5265ffa8/pyvista/utilities/reader.py#L306-L308

Reply:
Thank's for point that out, @adeak! I'll see if I can clean this up within this PR